### PR TITLE
Adding the 'kernel' topic

### PR DESCRIPTION
**Pull request by Pigmy-penguin**
### Thank you for contributing! Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [ ] Suggesting edits to an existing topic or collection
  - [x] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---------------------------------------------------------------------

### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288)
- [ ] Content (and my changes are in `index.md`)

> Please replace this line with an explanation of why you think these changes should be made.

### Curating a new topic or collection

- [x] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

> There are plenty of hobby kernels on github, and kernels are important pieces of software in everyone's life.

### Something that does not neatly fit into the binary options above

- [ ] My suggested edits are not about an existing topic or collection, or at least not a single one
- [ ] My suggested edits are not about curating a new topic or collection, or at least not a single one
- [ ] My suggested edits conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

> Please replace this line with an explanation of your proposed changes.

---------------------------------------------------------------------

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

File: topics/kernel/index.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+aliases: monolithic, microkernel, micro-kernel, exokernel, nanokernel
+display_name: Kernel
+short_description: Kernels are software at the core of modern operating systems that interact with hardware.
+topic: kernel
+wikipedia_url: https://en.wikipedia.org/wiki/Kernel_(operating_system)
+---
+A kernel is the heart of almost every operating system. It is always loaded in memory at any time and deals with the hardware to provide an interface for the software. It also manages peripherals, memory, interrupts, processes...
+Example of widely used kernels: Windows NT, Linux.
```
Comment (nholden) [inline topics/kernel/index.md:9]: What do you think about rewording this description to complete sentences?

```suggestion
A kernel is the heart of almost every operating system. It is always loaded in memory at any time and deals with the hardware to provide an interface for the software. It also manages peripherals, memory, interrupts, and processes. Examples of widely used kernels include Windows NT and Linux. 
```

File: topics/kernel/index.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+aliases: monolithic, microkernel, micro-kernel, exokernel, nanokernel
```
Comment (nholden) [inline topics/kernel/index.md:2]: `monolithic` could mean different things in different contexts, so I'm not sure it's an appropriate alias.

```suggestion
aliases: microkernel, micro-kernel, exokernel, nanokernel
```

**Review by nholden — COMMENTED** [review]
Thanks for this contribution, @Pigmy-penguin! It seems worthwhile to have a topic definition for `kernel`.

**Comment by Pigmy-penguin** [comment]
Excuse me, english is not my first language :)
Its perfect

**Review by kenyonj — APPROVED** [review]
💯 🍨 Thank you for your contribution! I'll merge this in as soon as CI is 🍏

**Pull request merged**